pkg/minishift/config/validations.go

```diff
@@ -31,18 +32,19 @@ import (
 
 	"github.com/minishift/minishift/pkg/minikube/constants"
 	minishiftConstants "github.com/minishift/minishift/pkg/minishift/constants"
-	"github.com/minishift/minishift/pkg/minishift/shell/powershell"
 	"github.com/minishift/minishift/pkg/util"
 	stringUtils "github.com/minishift/minishift/pkg/util/strings"
 )
 
 func IsValidDriver(string, driver string) error {
 	if runtime.GOOS == "windows" {
-		posh := powershell.New()
-		defer posh.Close()
-		checkWindowsName := `(Get-WmiObject -Class Win32_OperatingSystem).Caption`
-		stdOut, _ := posh.Execute(checkWindowsName)
-		if driver == "hyperv" && strings.Contains(stdOut, "Windows 7") {
+		cmd := exec.Command("wmic", "os", "get", "Caption", "/value")
+		stdOut, err := cmd.Output()
+		if err != nil {
+			return err
+		}
+		out := fmt.Sprintf("%s", stdOut)
+		if driver == "hyperv" && strings.Contains(out, "Windows 7") {
```

Comment:
Technically I could start Minishift on Windows XP, right?

Reply:
Yes, will also check if it is windows XP ;-)